test/test_charge_drift_models.jl

```diff
@@ -82,41 +82,109 @@ end
+    # test3: model both the bulk/inactive volumes
+    ## test 3.1: the bulk/inactive signals while varying the lifetimes: τ, τ_inactive
+    pos_bulk = CartesianPoint{T}(0.0085,0,0.005); Edep = 1u"eV"
+    @test !in(pos_bulk, simA_inactive_layer_geometry)
+    signalsum_list_bulk = []
+    τ_list = (10 .^ (-2:0.2:0))*u"ms"
+    pos_inactive = CartesianPoint{T}(0.0095,0,0.005); Edep = 1u"eV"
+    @test in(pos_inactive, simA_inactive_layer_geometry)
+    signalsum_list_inactive = []
+    τ_inactive_list = τ_list/100
+    for (τ,τ_inactive) in zip(τ_list, τ_inactive_list)
+        parameters = Dict("parameters" => Dict("τh" => τ, "τe" => τ, "τh_inactive" => τ_inactive, "τe_inactive" => τ_inactive),
+            "inactive_layer_geometry" => simA_inactive_layer_geometry)
+        trapping_model=ConstantLifetimeChargeTrappingModel{T}(parameters)
+        simA.detector = SolidStateDetector(simA.detector, trapping_model)
+        evt_bulk = Event(pos_bulk , Edep)
+        timed_simulate!(evt_bulk, simA)
+        signalsum_bulk  = T(0)
+        for i in 1:length(evt_bulk.waveforms)
+            signalsum_bulk  += abs(ustrip(evt_bulk.waveforms[i].signal[end]))
+        end
+        signalsum_bulk  *= inv(ustrip(SolidStateDetectors._convert_internal_energy_to_external_charge(simA.detector.semiconductor.material)))
+        push!(signalsum_list_bulk, signalsum_bulk)
+
+        evt_inactive = Event(pos_inactive , Edep)
+        timed_simulate!(evt_inactive, simA)
+        signalsum_inactive  = T(0)
+        for i in 1:length(evt_inactive.waveforms)
+            signalsum_inactive  += abs(ustrip(evt_inactive.waveforms[i].signal[end]))
+        end
+        signalsum_inactive  *= inv(ustrip(SolidStateDetectors._convert_internal_energy_to_external_charge(simA.detector.semiconductor.material)))
+        push!(signalsum_list_inactive, signalsum_inactive)
     end
-
+    @info signalsum_list_bulk
+    @test all(signalsum_list_bulk .< T(2.0))
+    @test all(diff(signalsum_list_bulk) .> 0)
+    @info signalsum_list_inactive
+    @test all(signalsum_list_inactive .< T(2.0))
+    @test all(diff(signalsum_list_inactive) .> 0)
+    @test all(signalsum_list_bulk .> signalsum_list_inactive)
+
+    ## test 3.2: the inactive layer signals while varying the depth
+    τ, τ_inactive = 1u"ms", 100u"ns"
+    parameters = Dict("parameters" => Dict("τh" => τ, "τe" => τ, "τh_inactive" => τ_inactive, "τe_inactive" => τ_inactive),
+        "inactive_layer_geometry" => simA_inactive_layer_geometry)
+    trapping_model=ConstantLifetimeChargeTrappingModel{T}(parameters)
+    simA.detector = SolidStateDetector(simA.detector, trapping_model)
+    signalsum_list_inactive = []
+    for depth in (0.1:0.1:0.9)/1000
+        pos_inactive = CartesianPoint{T}(0.01-depth,0,0.005); Edep = 1u"eV"
+        @test in(pos_inactive, simA_inactive_layer_geometry)
+        evt_inactive = Event(pos_inactive , Edep)
+        timed_simulate!(evt_inactive, simA, diffusion=false, Δt=1u"ns")
```

Comment:
No need to have explicit `diffusion = false`, this is the default
```suggestion
        timed_simulate!(evt_inactive, simA, Δt=1u"ns")
```